.github/workflows/docker-end-user.yml

```diff
@@ -8,6 +8,10 @@ on:
   #push:
   #  branches:
   #    - "**"
+  push:
+    # This will run every time a version tag is created
+    tags:
+      - 'v*'
```

Comment:
Are we going to use the `v` prefix? It does seem to be the custom on GitHub. @chrisrichardson?

Reply:
Good spot. The first release doesn't have a v. I guess we can actually just make this push for any tags

Reply:
I've remove the v prefix

Reply:
Well, I guess that's the reason to have `v`, to distinguish a release from any other tag. I think @chrisrichardson decided not to have `v` for a reason but I forgot why. 

On balance, I think we can trust people not to push tags.